tests/e2e/cucumber/features/integrations/link.feature

```diff
@@ -27,13 +27,27 @@ Feature: link
       | textfile.txt |
     #Then the public should see the following files on the files-drop page
     #  | textfile.txt |
-    #When the public reloads the public link pages
     #Then the public should not see the following files on the files-drop page
     #  | textfile.txt |
-    And "Anonymous" logs out
     When "Alice" downloads the following resources using the batch action
       | resource     | from         |
       | lorem.txt    | folderPublic |
       | textfile.txt | folderPublic |
     And "Alice" edits the public link named "myPublicLink" of resource "folderPublic" changing role to "editor"
+    And "Anonymous" refreshes the old link
+    And "Anonymous" downloads the following public link resources using the sidebar panel
```

Comment:
check whether `batch action` works or not.

Reply:
Just wondering how do you suggest to check that batch action works?  It checked here:`expect(download.suggestedFilename()).toBe('download.tar')`.

Reply:
@saw-jan as Viktor mentioned above we are asserting that the `zip` or `tar` file is downloaded in the when step itself.

Reply:
Yeah, saw that one :+1:
Just wanted to make sure that batch download in the public link also works but not to introduce the test step itself.

Reply:
The batch action for the public link doesn't work https://github.com/owncloud/web/issues/6423